### PR TITLE
Add scope field

### PR DESCRIPTION
**Pull request by lionick**
Run the following query to add the new field at cm_rcauth_sources
`alter table cm_rcauth_sources add scopes varchar(256)`

### CODE REVIEW

File: Console/Command/UpgradeVersionShell.php [hunk 1]
```diff
@@ -28,7 +28,8 @@ public function main() {
 
     public function _ug110()
     {
-        $query = "alter table cm_rcauth_sources alter column client_secret type varchar(1024) using client_secret::varchar(1024)";
+        $query = "alter table cm_rcauth_sources alter column client_secret type varchar(1024) using client_secret::varchar(1024);" .
+               "alter table cm_rcauth_sources add scopes varchar(256)";
```
[the remark this answers is not in the document]
Comment (lionick) [reply Console/Command/UpgradeVersionShell.php]: Done

[the remark this answers is not in the document]
Comment (lionick) [reply Console/Command/UpgradeVersionShell.php]: done

File: Console/Command/UpgradeVersionShell.php [hunk 2]
```diff
@@ -28,8 +28,23 @@ public function main() {
...
+        $dbc->begin();
+        try {
+            $this->RcauthSource->query($query);
+        }
+        catch(Exception $e) {
+            $dbc->rollback();
+            $err = true;
+            $this->out($e->getMessage());
+        }
+        if(!$err) {
```
Comment (ioigoume) [inline Console/Command/UpgradeVersionShell.php:44]: This is not needed. Add the commit in the try block. As soon as an error is thrown then the catch block will take control

File: Console/Command/UpgradeVersionShell.php [hunk 3]
```diff
@@ -28,8 +28,23 @@ public function main() {
 
     public function _ug110()
     {
-        $query = "alter table cm_rcauth_sources alter column client_secret type varchar(1024) using client_secret::varchar(1024)";
-        $this->RcauthSource->query($query);
-        $this->out('Change applied: ' . $query);
+        $err = false;
```
Comment (ioigoume) [inline Console/Command/UpgradeVersionShell.php:31]: Discard $err variable

File: webroot/css/style.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  }
+
+ul.form-list .field-info-flex .checkbox input {
+    width: auto;
+    flex-grow: 1;
+  }
+
+.checkbox {    
+    margin-left: 6px;
+}
```
Comment (ioigoume) [inline webroot/css/style.css:13]: Add a new line character here

**Comment by ioigoume** [comment]
@lionick rebase onto develop branch and resolve conflicts

File: View/RcauthSources/fields.inc [hunk 5]
```diff
@@ -119,16 +121,32 @@
...
+          <?php print _txt('pl.rcauthsource.issuer'); ?>
+          <span class="required">*</span>
+        </div>
+        <div class="field-desc"><?php print _txt('pl.rcauthsource.issuer.desc'); ?></div>
+    </div>
+    <div class="field-info">
+      <?php print ($e ? $this->Form->input('issuer', array('size' => 50, 'maxlength'=>'400')) : filter_var($rcauth_sources[0]['RcauthSource']['issuer'],FILTER_SANITIZE_SPECIAL_CHARS)); ?>
+    </div>
+  </li>
+  <li class="field-flex">
```
Comment (ioigoume) [inline View/RcauthSources/fields.inc:135]: Do we still need field-flex?

File: webroot/css/style.css [hunk 6]
```diff
@@ -0,0 +1,13 @@
+ul.form-list li.field-flex {
```
Comment (ioigoume) [inline webroot/css/style.css:1]: Do we need the style.css file now that we do not use the checkbox list

**Review by ioigoume — CHANGES_REQUESTED** [review]

**Comment by ioigoume** [comment]
@lionick do not forget to update the README.md file regarding the way the user should upgrade

**Review by ioigoume — APPROVED** [review]

**Pull request merged**